spec/index.html

```diff
@@ -3702,7 +3695,7 @@ <h3>ORDER BY</h3>
         </div>
         <p>The <a href="#op_lt">"&lt;" operator</a> (see the <a href="#OperatorMapping">Operator
             Mapping</a> and <a href="#operatorExtensibility">17.3.1 Operator Extensibility</a>) defines
-          the relative order of pairs of <code>numerics</code>, <code>simple literals</code>,
+          the relative order of pairs of <code>numerics</code>,
           <code>xsd:strings</code>, <code>xsd:booleans</code> and <code>xsd:dateTimes</code>. Pairs of
           IRIs are ordered by comparing them as <code>simple literals</code>.</p>
```

Comment:
What to do with this one?

Reply:
```suggestion
          IRIs are ordered by comparing them as <a data-cite="RDF12-CONCEPTS#dfn-simple-literal">simple literals</a>.</p>
```

Reply:
> What to do with this one?

It can be removed. It's covered by `xsd:string`.

Reply:
I'll do a full pass of the document to replace any (relevant) mentions of "simple literals".